src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -147,6 +150,9 @@ public AdyenConfigProperties(final Properties properties, final String currentRe
 
         this.acquirersList = properties.getProperty(PROPERTY_PREFIX + "acquirersList");
 
+        this.paymentProcessorAccountIdToMerchantAccount = properties.getProperty(PROPERTY_PREFIX + "paymentProcessorAccountIdToMerchantAccount");
```

Comment:
Could you update the `README` explaining how to use this? I'm actually quite confused what `paymentProcessorAccountId` refers to here.

Reply:
Done in [README.md](https://github.com/killbill/killbill-adyen-plugin/pull/95/files#diff-04c6e90faac2675aa89e2176d2eec7d8R44) file as below: 
```suggestion
* `org.killbill.billing.plugin.adyen.paymentProcessorAccountIdToMerchantAccount`: Mappings from the `paymentProcessorAccountId` to Adyen merchant accounts. The `paymentProcessorAccountId`, if exists in the plugin property, is a `String` set by the upstream logic to specify the merchant account used in the transaction.
```